docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-option-feature-integration.md

```diff
@@ -0,0 +1,465 @@
+
+
+### 1) Install the required modules using Composer
+
+1) Install the required modules:
+
+```bash
+composer require spryker-feature/marketplace-product-options:"dev-master" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure the following modules have been installed:
+
+| MODULE | EXPECTED DIRECTORY |
+|-|-|
+| MerchantProductOption | vendor/spryker/merchant-product-option |
+| MerchantProductOptionDataImport | vendor/spryker/merchant-product-option-data-import |
+| MerchantProductOptionGui | vendor/spryker/merchant-product-option-gui |
+| MerchantProductOptionStorage | vendor/spryker/merchant-product-option-storage |
+| MerchantProductOptionExtension | vendor/spryker/merchant-product-option-extension |
+
+{% endinfo_block %}
+
+
+### 2) Set up the database schema and transfer objects
+
+Adjust the schema definition so that entity changes will trigger events:
+
+**src/Pyz/Zed/PriceProductOffer/Persistence/Propel/Schema/spy_price_product_offer.schema.xml**
+```xml
+<?xml version="1.0"?>
+<database xmlns="spryker:schema-01" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed" xsi:schemaLocation="spryker:schema-01 https://static.spryker.com/schema-01.xsd" namespace="Orm\Zed\ProductOption\Persistence" package="src.Orm.Zed.ProductOption.Persistence">
+
+    <table name="spy_product_option_group" identifierQuoting="true">
+        <column name="key" type="VARCHAR" size="255" description="Key is an identifier for existing entities. This should never be changed."/>
+
+        <unique name="spy_product_option_group-unique-key">
+            <unique-column name="key"/>
+        </unique>
+    </table>
+</database>
+
+```
+
+Apply database changes and to generate entity and transfer changes:
+
+```bash
+console transfer:generate
+console propel:install
+console transfer:generate
+```
+
+
+{% info_block warningBox "Verification" %}
+
+Verify that the following changes have been implemented by checking your database:
+
+| DATABASE ENTITY | TYPE | EVENT |
+|-|-|-|
+| spy_merchant_product_option_group | table | created |
+| spy_product_option_group.key | column | created |
+
+Make sure that the following changes were applied in transfer objects:
+
+| TRANSFER  | TYPE  | EVENT | PATH |
+| - | - | - | - |
+| MerchantProductOptionGroup | class | created | src/Generated/Shared/Transfer/MerchantProductOptionGroupTransfer |
+| MerchantProductOptionGroupCriteria | class | created | src/Generated/Shared/Transfer/MerchantProductOptionGroupCriteriaTransfer |
+| MerchantProductOptionGroupCollection | class | created | src/Generated/Shared/Transfer/MerchantProductOptionGroupCollectionTransfer |
+| ProductOptionGroup.merchant | attribute | created | src/Generated/Shared/Transfer/ProductOptionGroupTransfer |
+
+{% endinfo_block %}
+
+### 3) Adjust Product Option Importer
```

Comment:
Please see https://spryker.atlassian.net/wiki/spaces/DOCS/pages/478118714/Feature+integration+guides for the allowed section names.